opentech/apply/activity/migrations/0029_migrate_old_submission_relation.py

```diff
@@ -6,12 +6,13 @@
 
 def submission_to_source(apps, schema_editor):
     Activity = apps.get_model('activity', 'Activity')
-    ContentType = apps.get_model('contenttypes', 'ContentType')
-    content_type = ContentType.objects.get(model='applicationsubmission', app_label='funds')
-    Activity.objects.update(
-        source_object_id=F('submission_id'),
-        source_content_type=content_type,
-    )
+    if Activity.objects.exists():
```

Comment:
Consider flipping this to a negative to fail fast:
```
if not Activity.objects.exists():
    return
```

It lowers the cognitive burden on the reader when considering the outcome of the conditional (a reader only has to look at one line vs 5).  You can rely on Activity objects existing after that check.

Alternatively drop it entirely, it only saves two database calls, one of which will a near no-op when there are no Activity rows.